src/args.rs

```diff
@@ -294,6 +294,12 @@ pub struct CliArgs {
     /// Enable README.md rendering in directories
     #[arg(long, env = "MINISERVE_README")]
     pub readme: bool,
+
+    /// Disable indexing
+    /// This will prevent miniserve from generating directory listings
+    /// and will return a 404 instead.
```

Comment:
```suggestion
    /// Disable indexing
    ///
    /// This will prevent directory listings from being generated
    /// and return an error instead.
```

Reply:
I don't see this being used. I'd prefer this wording.

Reply:
Ah whoops. I'll fix that.